src/h5web/metadata-viewer/AttributesTable.tsx

```diff
@@ -25,7 +34,21 @@ function AttributesTable(props: Props) {
         {attributes.map(({ name, value }) => (
           <tr key={name}>
             <th scope="row">{name}</th>
-            <td>{JSON.stringify(value)}</td>
+            <td>
+              {FOLLOWABLE_ATTRS.has(name) && typeof value === 'string' ? (
```

Comment:
I only deal with `string` attributes for now. Will deal with array of strings next.

Reply:
We might want to extract this in its own component then.